### ISSUE TITLE
False-positive in `double-slash-comment-whitespace-inside` rule

### ISSUE THREAD

**Issue by sindresorhus** [kristerkari/stylelint-scss#202]
This incorrectly causes a violation:

```scss
$bar: 'xmlns="http://bar"';
```

With stylelint-scss 2.3.0

**Comment by kristerkari** [comment]
Thanks, this should be easy to fix :+1:

**Comment by kristerkari** [comment]
@sindresorhus are you sure that that variable + string triggers a warning? I tried it both with a unit test and in a project, but I could not reproduce the bug.

**Comment by sindresorhus** [comment]
You're right. Seems it was affected by something else. I managed to reduce it to this:

```scss
$map: (
	'\'': '',
	'"': ''
);

$bar: 'xmlns="http://bar"';
```

**Comment by kristerkari** [comment]
Thanks, I'll try again using that :)

**Comment by kristerkari** [comment]
I had a new look at this, and it looks like the issue is that `postcss-scss` parser and `node-sass` both throw an error when trying to parse:

```scss
$map: (
	'\'': '',
	'"': ''
);
```

That then results, for some reason, to trigger a linting error for the next part:
```scss
$bar: 'xmlns="http://bar"';
```
